docsite/docs/dashboard/quickstart.mdx

```diff
@@ -0,0 +1,141 @@
+
+  </TabItem>
+
+  <TabItem value="load-balancer" label="By LoadBalancer" default>
+
+If your cluster supports LoadBalancer services, you can specify the creation of a LoadBalancer type service during the installation process with the following command.
+
+```bash
+helm install oceanbase-dashboard ob-operator/oceanbase-dashboard --version=0.2.1 --set service.type=LoadBalancer
+```
+
+If you have already installed OceanBase Dashboard, you can use the following command to modify the default service type to `LoadBalancer`. After the modification is successful, the cluster will assign an external IP to the OceanBase Dashboard service, and you can access the OceanBase Dashboard page through this external IP.
+
+```bash
+kubectl patch svc oceanbase-dashboard-oceanbase-dashboard --type=merge --patch='{"spec": {"type": "LoadBalancer"}}'
+```
+
+The External IP field will be assigned a value after the modification is successful.
+
+```bash
+$ kubectl get svc
+NAME                                      TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)        AGE
+oceanbase-dashboard-oceanbase-dashboard   LoadBalancer   192.168.xx.xx   xxx.xx.xxx.xx   80:xxxxx/TCP   1d5h
+```
+
+  </TabItem>
+
+  <TabItem value="port-forward" label="Temporary Access by Port Forwarding" default>
+
+If the node port of your cluster is not accessible so you cannot use the `NodePort` type service to expose OceanBase Dashboard, and your cluster does not support the `LoadBalancer` service, you can use the `kubectl port-forward` command to temporarily expose OceanBase Dashboard to a specified port on the current machine for temporary access. For example, executing the following command will expose OceanBase Dashboard to port 18081 on your current machine (the machine that executes this command).
+
+```bash
+kubectl port-forward -n default services/oceanbase-dashboard-oceanbase-dashboard 18081:80 --address 0.0.0.0
+```
+
+Visit `18081` port on your machine with a browser to open the login page. Specially, if the `port-forward` command is executed on your personal computer, you can access the service by opening a browser and visiting `http://127.0.0.1:18081`.
+
+  </TabItem>
+</Tabs>
+
+![Login Page](/img/dashboard/login.png)
+
+## Check Monitoring Metrics
+
+### Cluster Monitoring Metrics
+
+![Cluster Monitoring Metrics](/img/dashboard/cluster-metrics.png)
+
+### Tenant Monitoring Metrics
+
+![Tenant Monitoring Metrics](/img/dashboard/tenant-metrics.png)
+
+## Upcoming Features
+
+Besides the performance monitoring features for clusters and tenants, OceanBase Dashboard provides functions for cluster management, tenant management, backup management, and terminal connections to facilitate the operation and maintenance of OceanBase clusters. OBProxy management, monitoring alarms, SQL monitoring, and cluster diagnostics are also under active development and will be released in subsequent versions of the Dashboard.
```

Comment:
This part only need to show the upcoming features, "OBProxy management, monitoring alarms, SQL monitoring, and cluster diagnostics are also under active development and will be released in subsequent versions of the Dashboard."